plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -412,10 +412,15 @@ def checkCloudFlowIsPossible(self) -> None:
         active_machine = self._application.getMachineManager().activeMachine # type: Optional["GlobalStack"]
         if active_machine:
             
-            # Check 1: Printer isn't already configured for cloud
+            # Check 1A: Printer isn't already configured for cloud
             if ConnectionType.CloudConnection.value in active_machine.configuredConnectionTypes:
                 Logger.log("d", "Active machine was already configured for cloud.")
                 return
+            
+            # Check 1B: Printer isn't already configured for cloud
+            if active_machine.getMetaDataEntry("cloud_flow_complete", "value") is True:
+                Logger.log("d", "Active machine was already configured for cloud.")
+                return
 
             # Check 2: User did not already say "Don't ask me again"
             if active_machine.getMetaDataEntry("show_cloud_message", "value") is False:
```

Comment:
Same as above. The default of the metadata entry is False, right (and not "value")